templates/etc/systemd/system/kafka.service.j2

```diff
@@ -4,8 +4,8 @@ After=zookeeper.service
 
 [Service]
 Type=simple
-User=kafka
-Group=kafka
+User={{ kafka_user }}
+Group={{ kafka_group }}
 LimitNOFILE={{ nofiles_limit }}
 Restart=on-failure
 EnvironmentFile=/etc/default/kafka
```

Comment:
seems I missed the hardcoded user/group in this in the orig. review  .. and also the hardcoded kafka-bin-dir  
Line 12 below: `/usr/local/kafka` ?!